handlers/main.yml

```diff
@@ -1 +1,4 @@
-# Handlers for update_ca_trust
+- name: update-ca-trust
+  command: update-ca-trust force-enable
```

Comment:
What is "force-enable"? I don't see that command documented in the `update-ca-trust` manpage. I've always seen `extract` used as the command in these situations.

Reply:
That appears to be something I inherited from the RHEL6 update-ca-trust [man page](https://www.schloss.io/docsrv/man/centos6.5/man8/update-ca-trust.8/).  It appears deprecated in RHEL7+, so I've changed it to `extract`.